google/resource_bigtable_table.go

```diff
@@ -21,6 +21,12 @@ func resourceBigtableTable() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"family": {
```

Comment:
I think we can specify multiple column families here, so we can use a `schema.TypeSet`. As well, would you be able to change this to be a `schema.TypeSet` called `column_family` holding a `schema.TypeString` called `family`? [`CreateTableFromConf`](https://github.com/GoogleCloudPlatform/google-cloud-go/blob/master/bigtable/admin.go#L165) has a `GCPolicy` field that we don't represent yet, and don't need now - but I'd like to keep the door open for supporting it in the future.

So a config written as:

```hcl
resource "google_bigtable_table" "table" {
  name          = "my-table"
  instance_name = "${google_bigtable_instance.instance.name}"
  family        = "family-name"
}
```

would be written as:

```hcl
resource "google_bigtable_table" "table" {
  name          = "my-table"
  instance_name = "${google_bigtable_instance.instance.name}"
  column_family {
      family = "family-name"
  }
}
```

and I could specify multiple families like:


```hcl
resource "google_bigtable_table" "table" {
  name          = "my-table"
  instance_name = "${google_bigtable_instance.instance.name}"

  column_family {
      family = "family-name"
  }

  column_family {
      family = "other-family-name"
  }
}
```

Reply:
Seems legit, I will edit the code according to this